js/app.js

```diff
@@ -0,0 +1,104 @@
+import todoItem from './todoItem.js';
+import filterItem from './filterItem.js';
+let ID = 0;
+let DATA = []; // {id, context, complete}
+let HASH = "all";
+
+
+
+/************* element select & event setting ****************/ 
+const $todoInputElement = document.getElementById('new-todo-title');
+const $todoListElement = document.getElementById('todo-list');
+const $filtersElement = document.querySelector('.filters');
+const $countElement = document.querySelector('.todo-count strong')
+
+$todoInputElement.addEventListener('keyup', (event) => {
+	const key = event.key;
+	const valueTrim = event.target.value.trim()
+
+    if (key === "Enter" && valueTrim) {
+        let data = {
+            id: ID++,
+            context: valueTrim,
+            complete: false
+		}
+		inputEvent(data)
+		render()
```

Comment:
```suggestion
		inputEvent({
		    id: ID++,
		    context: valueTrim,
		    complete: false
		})
		render()
```
선언하지 말고 바로 넘겨줘도 좋을 것 같습니다!